src/components/deposit/Confirming.js

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect } from 'react'
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+
+import { autoSubmitDepositProof } from '../../actions'
+import StatusIndicator from '../svgs/StatusIndicator'
+
+import { BitcoinHelpers } from '@keep-network/tbtc.js'
+
+import BigNumber from "bignumber.js"
+BigNumber.set({ DECIMAL_PLACES: 8 })
+
+const Confirming = ({ autoSubmitDepositProof, signerFeeInSatoshis }) => {
+  useEffect(() => {
+    autoSubmitDepositProof()
+  }, [autoSubmitDepositProof])
```

Comment:
It does, but I didn't see any issue when testing the flow when it transitioned from `/pay` to `/confirming`. Still, I'll try adding some conditional to prevent it running unnecessarily!

Reply:
Yeah, because tbtc.js doesn't re-submit in those cases it should be fine, but I'm worried if we change that for whatever reason it could lead to some pretty hard to trace outcomes.